<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentMange</class>
 <widget class="QMainWindow" name="StudentMange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-10</y>
      <width>271</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2">
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>上午</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>下午</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="comboBox_3">
       <item>
        <property name="text">
         <string>401</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>402</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>403</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>日期</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>45</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>46</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>下周一</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>下周二</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>下周三</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>下周四</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>下周五</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>下周六</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>下周日</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>预约确认</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>时段</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>自习室</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>531</width>
      <height>331</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>270</y>
       <width>503</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>查看自习室容量</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>511</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>601</width>
      <height>210</height>
     </rect>
    </property>
    <widget class="QTableView" name="tableView_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>581</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>390</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>查看预约记录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>470</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>839</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
